src/app/api/[[...route]]/routes/task.route.ts

```diff
@@ -9,10 +9,46 @@ const app = new Hono();
 
 export default app;
 
-app.patch('/:taskId', (c) => {
-  const taskId = c.req.param('taskId');
-  return c.json({ message: `Updating task id: ${taskId}` });
-});
+app.patch(
+  '/:taskId',
+  zValidator('json', taskCreationSchema.omit({ categoryId: true }).partial()),
```

Comment:
I'd like you to define `taskCreationSchema.omit({ categoryId: true }).partial()` in the `src/constants/schema.ts` and use it by importing it from there. This makes it possible to reuse it in the frontend.

Reply:
I've created a new schema for task update